### DISCUSSION TITLE
Why is the PHOTO drive set to read-only?

### DISCUSSION THREAD

**Discussion by freddieFishCake** [JarvyJ/pislide-os#16] [Q&A] [answered]
On first use I added a few images to a folder in the PHOTO drive. All went well ;-)
Subsequently I wanted to add more images and captions, but the folder I made in PHOTO is set to read-only :-(

**Comment by JarvyJ** [comment, marked as answer]
Hmmm, the `PHOTO` partition always gets loaded as read/write, since that's where the cached images go if needed. What Pi are you using? Also is there any chance it's the SD Card "Lock" slider?

**Comment by freddieFishCake** [comment]
Agreed, the problem turned out to be a faulty write protect slider on the SD card adapter I was using .. sorry about that ;-)
